packages/ngspice/build.sh

```diff
@@ -1,10 +1,45 @@
-TERMUX_PKG_HOMEPAGE=http://ngspice.sourceforge.net/
+TERMUX_PKG_HOMEPAGE=https://ngspice.sourceforge.net/
 TERMUX_PKG_DESCRIPTION="A mixed-level/mixed-signal circuit simulator"
 TERMUX_PKG_LICENSE="BSD 3-Clause, LGPL-2.1"
 TERMUX_PKG_LICENSE_FILE="COPYING"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=35
+TERMUX_PKG_VERSION=37
 TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/ngspice/ngspice-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=c1b7f5c276db579acb3f0a7afb64afdeb4362289a6cab502d4ca302d6e5279ec
-TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_SHA256=9beea6741a36a36a70f3152a36c82b728ee124c59a495312796376b30c8becbe
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--enable-xspice
+--enable-cider
+--with-readline=yes
+--enable-openmp
+"
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_EXTRA_HOSTBUILD_CONFIGURE_ARGS="
+--with-x=no
+--enable-cider
+--enable-xspice
+"
+TERMUX_PKG_DEPENDS="libc++, readline, fftw"
 TERMUX_PKG_GROUPS="science"
+
+termux_step_host_build(){
+	cp -r $TERMUX_PKG_SRCDIR/* .
```

Comment:
Looks like host build step can be simplified to just something like:

```
$TERMUX_PKG_SRCDIR/configure $TERMUX_PKG_EXTRA_HOSTBUILD_CONFIGURE_ARGS
cd src/xspice/cmpp && make
```

Reply:
Let me test again those deb first before resolving this.